### ISSUE TITLE
Duboku error 

### ISSUE THREAD

**Issue by jaajaajaajann** [groggyegg/plugin.video.duboku#24]
Looks like there is an error again running duboku , updated the ResolveURL and still getting an error

**Comment by rainbatt** [comment]
Yes...seems so.  Unable to access

**Comment by jaajaajaajann** [comment]
Yup, able to view the list of shows and add them and see episodes but just won't play.

**Comment by gamer9394** [comment]
I gave up and using web version now.....

**Comment by kin2son** [comment]
Anyone has any update/solution of fixing Duboku Kodi addon?

**Comment by jaajaajaajann** [comment]
I think we need to wait for @groggyegg to have time to look into it.

**Comment by kin2son** [comment]
I've actually managed to fix it on my Google TV by re-installing the add on and applying the zip fix that was shared a few months ago

**Comment by jaajaajaajann** [comment]
Which zip file?

**Comment by jaajaajaajann** [comment]
Good suggestion, v. 1.0.4 worked

**Comment by gamer9394** [comment]
Is there a way to download it?

**Comment by jaajaajaajann** [comment]
> Is there a way to download it?

I had this previously downloaded and saved in my Kodi. @groggyegg will need to share the Google drive link/access or have time to look into the new issue. Or maybe @kin2son can direct you to the link he used.

**Comment by kimton2007** [comment]
it work now thank you so much.> 

i did download but still no luck

**Comment by jhleong75** [comment]
Yes not working. I think can give up since not many use it. 

> > 
> 
> i did download but still no luck

**Comment by jaajaajaajann** [comment]
Worked when I loaded 1.0.4. and still working 

[repository.dnb-kodi-repo-1.0.4.1.zip](https://github.com/user-attachments/files/16676029/repository.dnb-kodi-repo-1.0.4.1.zip)

**Comment by kimton2007** [comment]
thank you working good

**Comment by jhleong75** [comment]
Thanks this one work now. 

> Worked when I loaded 1.0.2. and still working
> 
> [repository.dnb-kodi-repo-1.0.4.1.zip](https://github.com/user-attachments/files/16676029/repository.dnb-kodi-repo-1.0.4.1.zip)

**Comment by rainbatt** [comment]
Is this a new addon for Duboku?

**Comment by groggyegg** [comment]
I think I might deprecate the addon since I don’t think I want to spend too much time on this. Alternative app is to use https://adrama.app/

**Comment by kimton2007** [comment]
> 

Hi Groggyegg can i use this for firetv ?

**Comment by groggyegg** [comment]
> > 
> 
> Hi Groggyegg can i use this for firetv ?

Yes, any android tv should work just follow the instructions https://adrama.app/install

**Comment by kimton2007** [comment]
Hi Groggyegg sorry I mean firestick
Thank you

**Comment by kimton2007** [comment]
Thank you Groggyegg I did it work now

**Comment by kimton2007** [comment]
Hi jhleong75
duboku still work my it down
https://github.com/jhleong75)Worked when I loaded 1.0.2. and still working

[repository.dnb-kodi-repo-1.0.4.1.zip](https://github.com/user-attachments/files/16676029/repository.dnb-kodi-repo-1.0.4.1.zip)
